src/pages/dho/Explore.vue

```diff
@@ -186,13 +211,18 @@ q-page.page-explore
           q-btn.q-px-lg.h-btn1(no-caps rounded unelevated color="secondary" href="https://hypha.earth/" target="_blank") Discover More
 
   .row.q-py-md
-    .col-sm-12.col-md-12.col-lg-9(ref="scrollContainer")
+    .col-sm-12.col-md-12.col-lg-9(ref="scrollContainer" v-if="exploreBy === 'DAOS'")
       q-infinite-scroll(@load="onLoad" :offset="250" :scroll-target="$refs.scrollContainer" ref="scroll")
         .row
           .col-4.q-mb-md(v-for="(dho,index) in dhos" :key="dho.name" :class="{ 'col-6': $q.screen.lt.lg, 'q-pr-md': $q.screen.lt.sm ? false : $q.screen.gt.md ? true : index % 2 === 0, 'full-width':  view === 'list' || $q.screen.lt.sm}")
             dho-card.full-width(v-bind="dho" :view="view")
-
+    .col-9(v-if="exploreBy === 'ECOSYSTEMS'")
```

Comment:
@arsenijesavic It is still using strings here ( `ECOSTSTENS` ) please check everywhere